catroid/config/lint-baseline.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
   ~ Catroid: An on-device visual programming system for Android devices
-  ~ Copyright (C) 2010-2020 The Catrobat Team
+  ~ Copyright (C) 2010-2021 The Catrobat Team
```

Comment:
Please check if changing the `lint-baseline.xml` file is really necessary after adding .png files for different resolutions.